core/src/test/scala/com/databricks/labs/remorph/transpilers/TsqlToDatabricksTranspilerTest.scala

```diff
@@ -0,0 +1,22 @@
+package com.databricks.labs.remorph.transpilers
+
+import org.scalatest.wordspec.AnyWordSpec
+
+class TsqlToDatabricksTranspilerTest extends AnyWordSpec with TranspilerTestCommon {
+
+  protected final val transpiler = new TSqlToDatabricksTranspiler
+
+  "The TSQL-to-Databricks transpiler" when {
+    "transpiling set operations" should {
+      val expectedTranslations = Map(
+        "(SELECT a, b from c) UNION SELECT x, y FROM z" -> "(SELECT a, b from c) UNION (SELECT x, y from z);",
+        "(SELECT a, b from c) UNION ALL SELECT x, y FROM z" -> "(SELECT a, b from c) UNION ALL (SELECT x, y from z);",
+        "(SELECT a, b FROM c)" -> "SELECT a, b FROM c;")
+      expectedTranslations.foreach { case (originalTSql, expectedDatabricksSql) =>
+        s"correctly transpile: $originalTSql" in {
+          originalTSql transpilesTo expectedDatabricksSql
+        }
+      }
+    }
+  }
```

Comment:
@sundarshankar89: is this what you had in mind?

Reply:
under tests/resources/functional/tsql/core_engine/union/

`union_simple.sql`
```
-- tsql sql:
(SELECT a, b from c) UNION SELECT x, y FROM z

-- databricks sql:
(SELECT a, b from c) UNION (SELECT x, y from z);
```
`union_all_simple.sql`
```
-- tsql sql:
(SELECT a, b from c) UNION ALL SELECT x, y FROM z

-- databricks sql:
(SELECT a, b from c) UNION  ALL (SELECT x, y from z);
```

Reply:
@sundarshankar89: Implemented, except in the folder that the python implementation also tests against. (The python implementation already supported this, so the test in that location guards against regressions.)